### PR TITLE
Use Scala3 on this tutorial

### PR DESCRIPTION
**Pull request by FabioPinheiro**
What are the guidelines to update the tutorials repositories are you going to have two versions (one for scala2 and another scala3)?

### CODE REVIEW

File: scalajs-tutorial-fastopt.html [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
-</html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>The Scala.js Tutorial</title>
+</head>
+
+<body>
+  <!-- Include Scala.js compiled code -->
+  <script type="text/javascript" src="./target/scala-3.0.0/scala-js-tutorial-fastopt/main.js"></script>
```
Comment (FabioPinheiro) [inline scalajs-tutorial-fastopt.html:11]: I was expecting something like "./target/scala-3.0" instead of "./target/scala-3.0.0"
Previously the minor version did not appear on that target folder path. @sjrd was this change intentional?

Comment (sjrd) [reply scalajs-tutorial-fastopt.html:11]: This entirely depends on the default value of sbt's `crossTarget` setting, so you'd have to ask that to sbt.

**Comment by sjrd** [comment]
Thanks for the PR. However, we never merge PRs in this repository because the *commit history* accompanies the tutorial on the website.

Please open an issue at https://github.com/scala-js/scala-js-website/issues instead.
